tests/test_api_versions.py

```
@@ -109,19 +109,17 @@ def test_rest_framework_versioning_config(self, settings: object) -> None:
 @pytest.mark.skipif(not HAS_DRF, reason="DRF not installed in test environment")
 class TestVersionEndpoints:
     def test_versions_list_empty_ok(self) -> None:
-        from uuid import uuid4
         client = APIClient()
-        resp = client.get(f"/api/versions/Patient/{uuid4()}/")
+        resp = client.get(f"/api/versions/Patient/999999/")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)